WaterNetworkManagement/Simulation/Result/schema.json

```diff
@@ -0,0 +1,48 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "$id": "https://smart-data-models.github.io/data-models/specs/WaterNetwork/Result/schema.json",
+    "title": "WaterNetwork - Result",
+    "description": "This entity contains a harmonised description of a generic simulation result made for the Water Network Management domain. This entity is primarily associated with the water network management vertical and related IoT applications.",
+    "type": "object",
+    "allOf": [
+        {
+            "$ref": "https://smart-data-models.github.io/data-models/common-schema.json#/definitions/Location-Commons"
+        },
+        {
+            "$ref": "https://raw.githubusercontent.com/smart-data-models/dataModel.WaterNetworkManagement/master/WaterNetworkManagement-schema.json"
+        },
+        {
+            "properties": {
+                "type": {
+                    "type": "string",
+                    "enum": [
+                        "Result"
+                    ],
+                    "description": "NGSI-LD Entity Type"
+                },
+                "description": {
+                    "$ref": "https://forge.etsi.org/rep/NGSI-LD/NGSI-LD/raw/master/schema/Entity.json#/definitions/Property"
+                },
+                "hasInputNetwork": {
+                    "$ref": "https://forge.etsi.org/rep/NGSI-LD/NGSI-LD/raw/master/schema/Entity.json#/definitions/Relatnioship"
+                },
+                "hasSimulation": {
```

Comment:
I don't think `has` is the right verb to use for a relationship in this direction - I'd say a simulation scenario has a result rather than a result has a simulation scenario. A result is *from* a simulation scenario.

Reply:
refSimulationScenario maybe

Reply:
I prefer this.